demos/kitchen_sink/screens_data.json

```diff
@@ -97,15 +97,6 @@
       'object': 0,
       'icon': 'dock-window'
   },
-  'User Animation Card': {
-      'kv_string': 'user_animation_card',
-      'Import': 'from libs.baseclass.user_animation_card import KitchenSinkUserCard',
-      'Factory': 'KitchenSinkUserCard()',
-      'name_screen': 'user animation card',
-      'source_code': 'Components-User-Animation-Card.md',
-      'object': 0,
-      'icon': 'animation'
-  },
```

Comment:
the widget UserAnimationCard is no longer part of the kivymd library, thus trying to import it will only crash the demo app